extensions/amp-share-tracking/0.1/test/test-amp-share-tracking.js

```diff
@@ -31,6 +31,7 @@ describe('amp-share-tracking', () => {
     sandbox = sinon.sandbox.create();
     viewerForMock = sandbox.stub(Viewer.prototype, 'getFragment');
     xhrMock = sandbox.stub(Xhr.prototype, 'fetchJson');
+    sandbox.stub(Math, 'random', () => 0.111111111111111111111111111111);
```

Comment:
nit: you can put some random numbers at the end, to make it clear that we truncate the string